setuptools/unicode_utils.py

```diff
@@ -4,6 +4,8 @@
 from setuptools.extern import six
 
 # HFS Plus uses decomposed UTF-8
+
```

Comment:
I think this change is wrong. In my experience, the comments should be adjacent to their relevant code. These additional new lines make the comment appear as a standalone construct.